src/devices/Qmc5883l/Qmc5883l.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Device.Model;
+using System.Numerics;
+
+namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// 3-Axis Digital Compass QMC5883L
+    /// </summary>
+    [Interface("3-Axis Digital Compass QMC5883L")]
+    public class Qmc5883l : IDisposable
+    {
+        /// <summary>
+        /// QMC5883L Default I2C Address
+        /// </summary>
+        public const byte DefaultI2cAddress = 0x0D;
+
+        /// <summary>
+        /// QMC5883L Chip id.
+        /// </summary>
+        public const byte Chip_Id = 0xff;
```

Comment:
Same with the const and anything public and private, the same naming convention applies, so `ChipId` or `ChipIdentifier` as we prefer not to use abbreviation